src/core/src/sys/interactions.rs

```diff
@@ -443,4 +445,17 @@ mod test {
         interactions.add_global(Box::new(Wolf::new(1.0)));
         assert_eq!(interactions.globals().len(), 1);
     }
+
+    #[test]
+    fn test_globals_cutoff() {
```

Comment:
I think we mainly need to get the largest cutoff. Let's use the [YAGNI](https://en.wikipedia.org/wiki/You_aren%27t_gonna_need_it) principle!